linkcheck/url.py

```diff
@@ -60,20 +60,23 @@
     "_hex_full": r"0-9a-f",
     "_part": r"([a-z0-9][-a-z0-9]{0,61}|[a-z])",
 }
-_safe_char = r"([a-z0-9%(_path)s\+]|"\
-             r"(%%[%(_hex_safe)s][%(_hex_full)s]))" % _basic
+_safe_char = r"([a-z0-9%(_path)s\+]|" r"(%%[%(_hex_safe)s][%(_hex_full)s]))" % _basic
```

Comment:
No.  This makes it worse.  Especially the concatenation.

Reply:
To clarify my position:
```suggestion
_safe_char = (
    r"([a-z0-9%(_path)s\+]|"
    r"(%%[%(_hex_safe)s][%(_hex_full)s]))" % _basic
)
```

Reply:
Done.